topo/node/ixia/ixia.go

```diff
@@ -81,6 +83,31 @@ func (n *Node) CreateNodeResource(ctx context.Context, ni node.Interface) error
 	return nil
 }
 
+func (n *Node) GetNodeResourceStatus(ctx context.Context, ni node.Interface) (*node.NodeStatus, error) {
```

Comment:
This should probalbly just be 

Status(ctx)

node now provides the node.Interface so you don't need to pass it in anymore

Reply:
Will update.